<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Otter::BookmarksImporterWidget</class>
 <widget class="QWidget" name="Otter::BookmarksImporterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="removeCheckBox">
     <property name="text">
      <string>Remove existing bookmarks</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="removeDependentStackedWidget">
     <widget class="QWidget" name="removePage">
      <layout class="QFormLayout" name="formLayout_3">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="1" column="0">
        <widget class="QCheckBox" name="notDuplicateCheckBox">
         <property name="text">
          <string>Do not duplicate bookmarks with same address</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <layout class="QFormLayout" name="removeForm">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="folderLabel">
           <property name="text">
            <string>Import into folder:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="folderComboBox"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="noRemovePage">
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QCheckBox" name="toSubfolderCheckBox">
         <property name="text">
          <string>Import into subfolder</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QFrame" name="noRemoveFrame">
         <layout class="QFormLayout" name="noRemoveBox">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="subfolderNameLabel">
            <property name="text">
             <string>Subfolder name:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="subfolderNameLineEdit">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Leave empty to import into main folder&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
